docs/contribute.rst

```diff
@@ -1,5 +1,3 @@
-.. _contributing-to-read-the-docs:
```

Comment:
Be careful when removing these ones. This one in particular, is used at https://github.com/rtfd/readthedocs.org/pull/3624/files#diff-c7f13470cacdf0634e185d4313d02a12R55 (L57 in `design.rst`) so that link is probably dead after removing this.

Reply:
Ups, I just missed that one. I usually do a grep to search those references.